server/db/models/Product.js

```diff
@@ -0,0 +1,49 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Product = db.define('product', {
+  title: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  author: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  description: {
+    type: Sequelize.TEXT,
+  },
+  summary: {
+    type: Sequelize.TEXT,
+    allowNull: false,
+  },
+  format: {
+    type: Sequelize.ENUM('hardcover', 'paperback', 'ebook'),
+    defaultValue: 'hardcover',
+    allowNull: false,
+  },
+  ISBN13: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  imageUrl: {
+    type: Sequelize.STRING,
+    //defaultValue: find default photo for public folder
+  },
+  price: {
+    type: Sequelize.DECIMAL,
+    allowNull: false,
+  },
+  length: {
+    type: Sequelize.INTEGER,
```

Comment:
Should we make this a string so it can say "200 pages" vs. "5 hours" in the case of an audio book?
Also wondering if we should allow null in case we ned up having products without a length